backupfile/ubuntu18.json

```diff
@@ -0,0 +1,28 @@
+{
+    "variables": {
+        "machine_type": "random" 
```

Comment:
Looks like you commit some unnecessary files (temporary, backup and so on) I guess it should be removed